### ISSUE TITLE
Crafting new visualization techniques for the mobile web

### ISSUE THREAD

**Issue by mmmavis** [mozilla/mozfest-program#458]
**[ Google Spreadsheet Row Number ]** 432
**[ Facilitator ]** Aaron Williams
### Description

It's time we treat the mobile web as first class and desktop experiences as secondary. 

In this session, we will discuss and design data visualization techniques that are mobile specific. Rather than design visuals for desktop that scale down, we'll design for mobile and scale up. Mobile interaction is an open field, so how can we use tap, shake, and rotate as tools for storytelling and visualization?

Let's think about how both the mobile web and the devices themselves can illustrate data in ways a desktop browser or app cannot.
### Agenda

This session will be part chat and part workshop. First, we'll discuss the challenges of presenting data on the mobile web. We'll think about common data visualization patterns and discuss what new ones we could create.

Then, we'll break into groups where each one will take a pattern or visualization and design it as a mobile-first experience. Given time, we'll look to implement working prototypes.
### Participants

The session is quite flexible. If we have <=5 members, our group will do a deep dive into a specific data visualization technique and attempt to evolve it for the mobile web. As the number of participants grow, we can tackle more data visualization styles.   
### Outcome

I hope that participants will begin to think about their data visualization experiences as mobile experiences. I want the group to act as visualization explorers who will continue to explore the uncharted territory of mobile data visualization.

**Comment by ryanpitts** [comment]
Aaron would need travel support to be able to attend MozFest.

**Comment by ryanpitts** [comment]
Journalism space is accepting this session, and adding to the "Mobile & Accessibility" pathway.
